v1/pipeline/steps/cd/k8s-scale-step-info.yaml

```diff
@@ -3,48 +3,18 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - type: object
   required:
-  - instanceSelection
-  - workload
+    - instanceSelection
```

Comment:
should follow snake case for field name instanceSelection -> instance_selection

Reply:
done